src/puzzle/Bank.js

```diff
@@ -213,7 +213,7 @@ pzpr.classmgr.makeCommon({
 		setData: function(value, index) {
 			var len = this.board.bank.pieces.length;
 			if (index < 0 || index > len) {
-				throw "Index out of range";
+				throw Error("Index out of range");
```

Comment:
Shouldn't this be `throw new Error` instead?

Reply:
`throw Error` and `throw new Error` are identical in behavior per the ES spec: https://262.ecma-international.org/7.0/#sec-error-constructor

You could argue the latter is better on principle but the code seemed to use the former in a few places already.